src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue

```diff
@@ -0,0 +1,284 @@
+      };
+      if (this.isStickyBottomLayout) {
+        style.marginTop = '-52px';
+        style.marginLeft = '-10px';
+        style.maxWidth = '400px';
+      } else {
+        style.maxWidth = '480px';
+      }
+      return style;
+    },
+    identitySlotProps() {
+      return {
+        x: 140,
+        y: 57,
+      };
+    },
+    saveSlotProps() {
+      return {
+        x: 148,
+        y: 72,
+      };
+    },
+    labelY() {
+      return 140;
+    },
+    textStyle() {
+      return {
+        color: '#9B9B9B',
+        fontFamily: 'Source Sans Pro, Arial, sans-serif',
+        fontSize: '14px',
+      };
+    },
+    labelStyle() {
+      return {
+        ...this.textStyle,
+        textAlign: 'center',
+        width: '100%',
+      };
+    },
+    labelButtonStyle() {
+      return {
+        padding: '0 2px',
+        border: 'none',
+        borderRadius: '4px',
+        outline: 'none',
+        color: 'inherit',
+        fontFamily: 'inherit',
+        fontSize: 'inherit',
+        transition: 'background-color 0.2s ease',
+      };
+    },
+    hintLabelStyle() {
+      return {
+        ...this.textStyle,
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        textAlign: 'left',
+        alignItems: 'flex-end',
+      };
+    },
+    ctaButtonClass() {
+      return [
+        'likecoin-button-widget__cta-button',
+        `likecoin-button-widget__cta-button--${this.ctaButtonPreset}`,
+      ];
+    },
+    ctaHref() {
+      return CLW3_NOTICE_URL;
+    },
+  },
+};
+</script>
+
+<style lang="css">
+.likecoin-button-widget__label-button {
+  background-color: transparent;
+}
+.likecoin-button-widget__label-button:not([disabled]):hover {
+  background-color: #f4f4f4;
+}
+.likecoin-button-widget__label-button:not([disabled]):active {
+  background-color: #e6e6e6;
+}
+
+.likecoin-button-widget__cta-button {
+  display: block;
+
+  box-sizing: border-box;
+
+  width: max-content;
+  height: 36px;
+  padding: 4px 36px;
+
+  transition-timing-function: ease;
+  transition-duration: 0.2s;
+  transition-property: border-color, background-color;
+
+  text-decoration: none;
+
+  color: #28646e;
+  border-radius: 12px;
+  outline: none;
+
+  font-size: 14px;
+  font-weight: 600;
+}
+.likecoin-button-widget__cta-button.likecoin-button-widget__cta-button--default {
+  border: 3px solid #aaf1e7;
+  background-color: white;
+}
+.likecoin-button-widget__cta-button.likecoin-button-widget__cta-button--default:hover {
+  border-color: #50e3c2;
+}
+.likecoin-button-widget__cta-button.likecoin-button-widget__cta-button--default:active {
+  background-color: #d2f0f0;
+}
+
+.likecoin-button-widget__cta-button.likecoin-button-widget__cta-button--special {
+  border: none;
+  background: linear-gradient(78deg, #d2f0f0, #f0e6b4);
+}
+.likecoin-button-widget__cta-button.likecoin-button-widget__cta-button--special:hover {
+  background: linear-gradient(78deg, #cae6e6, #e9dfae);
+}
+</style>
```

Comment:
It seems the above CSS is not scoped, perhaps can be moved to global CSS to share with the old version?